gradle/libs.versions.toml

```diff
@@ -18,11 +18,15 @@ espressoCore = { module = "androidx.test.espresso:espresso-core", version.ref =
 espressoWeb = { module = "androidx.test.espresso:espresso-web", version.ref = "espresso" }
 testCore = "androidx.test:core:1.3.0"
 uiAutomator = "androidx.test.uiautomator:uiautomator:2.2.0"
+fragmentTesting = "androidx.fragment:fragment-testing:1.3.2"
+robolectric = "org.robolectric:robolectric:4.5.1"
 
-androidXCore = "androidx.core:core:1.3.1"
+androidXCore = "androidx.core:core:1.3.0"
 androidXRules = "androidx.test:rules:1.3.0"
+androidXTest = "androidx.test.ext:junit:1.1.2"
+androidXTestKtx = "androidx.test.ext:junit-ktx:1.1.2"
 
-appcompat = "androidx.appcompat:appcompat:1.2.0"
+appcompat = "androidx.appcompat:appcompat:1.3.0"
```

Comment:
the versions needed to be adjusted, otherwise we are including one of the libraries with 2 different versions, what throws an error while running tests with Robolectric on the JVM